src/pages/index.vue

```diff
@@ -60,19 +60,24 @@
           <h2 class="dashboard__heading">
             State Channels
           </h2>
-          <p class="dashboard__paragraph">
+        </div>
+        <div class="dashboard__column">
+          <p class="dashboard__paragraph dashboard__paragraph--horizontal">
             State Channels allow the gas-free execution of smart contracts and
             transactions, privately and with the speed of light, while still
             being able to escalate on-chain in case of disagreement.
             <app-link
               class="dashboard__link"
-              to="https://aeternity.com/state-channels">
+              to="https://medium.com/aeternity-crypto-foundation/aeternity-bitcoin-ng-the-way-it-was-meant-to-be-df7bb1d65a4b"
```

Comment:
The old one is correct, it's about state channels.

Reply:
reverted